components/camel-webhook/src/main/java/org/apache/camel/component/webhook/WebhookEndpoint.java

```diff
@@ -78,10 +79,16 @@ public void afterPropertiesConfigured(CamelContext camelContext) {
         delegateEndpoint.setWebhookConfiguration(configuration);
     }
 
+    @Override
+    protected void doInit() throws Exception {
+        super.doInit();
+        ServiceHelper.initService(delegateEndpoint);
```

Comment:
This might be not necessary here. And it should be in https://github.com/apache/camel/blob/master/core/camel-base/src/main/java/org/apache/camel/impl/engine/AbstractCamelContext.java#L1457